### DISCUSSION TITLE
Add a link in Team > About

### DISCUSSION THREAD

**Discussion by jd-imi** [community/community#140180] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there a way to add a link in the "About" section of a team page (`https://github.com/orgs/<MyOrganisation>/teams/<MyTeam>`) ?

**Comment by tshandler** [comment]
No, GitHub does not natively support adding clickable links in the "About" section of a team's page within an organization. The text entered in the "About" section is plain text, and any URLs added there will not be automatically converted to clickable links.

However, you can work around this by:
1. Including the full URL in the "About" section as plain text, so team members can manually copy and paste it into their browser.
2. Alternatively, add the link in other team communications or documentation, like the team’s README file or GitHub Discussions, where Markdown is supported and allows for clickable links.
